docs/source/additional/release.rst

```diff
@@ -31,6 +31,23 @@ This is an overview of the main changes, please have a look at the changelog of
 - `backend changelog <https://github.com/Substra/substra-backend/blob/main/CHANGELOG.md>`__
 - `orchestrator changelog <https://github.com/Substra/orchestrator/blob/main/CHANGELOG.md>`__
 
+Substra 0.38.0 --- 2024-06-13
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**All**
+
+- The release 0.38.0 focuses on improving the security and limit the rights of the differents Substra pods when deployed on a cluster. The following changes have been made:
+
+  - Ressources requests and limits are now set as default for the backend and frontend.
```

Comment:
Did we not set default values for the orchestrator?

Reply:
It was already all set for the orc if I'm not mistaken

Reply:
It seems like it was introduced in chart version [8.5.0](https://github.com/Substra/orchestrator/blob/main/charts/orchestrator/CHANGELOG.md#850---2024-04-05)

Substra 0.37.0 targets chart version [8.4.0](https://github.com/Substra/substra-documentation/blob/main/docs/source/additional/releases.yaml#L32)

Reply:
Also I think I would reword a bit:
```suggestion
  - CPU and memory requests and limits are now provided by default for all orchestrator, backend and frontend pods.
```

Reply:
Ok nice catch thanks !!